service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ComplyWithDirectionsControllerTest.java

```diff
@@ -43,6 +44,51 @@
         super("comply-with-directions");
     }
 
+    @Test
+    void shouldCafcassDirectionToCafcassCollectionWhenCourtRespondsOnBehalfOf() {
+        UUID uuid = randomUUID();
+
+        DirectionResponse responseForCafcass = DirectionResponse.builder()
+            .directionId(uuid)
+            .assignee(COURT)
+            .complied("Yes")
+            .respondingOnBehalfOf("CAFCASS")
+            .build();
+
+        DirectionResponse responseForCourt = DirectionResponse.builder()
+            .directionId(uuid)
+            .assignee(COURT)
+            .complied("Yes")
+            .build();
```

Comment:
Just want to get your opinion, is this okay?

```suggestion
        DirectionResponse.DirectionResponseBuilder directionResponseBuilder = DirectionResponse.builder()
            .directionId(uuid)
            .assignee(COURT)
            .complied("Yes");

        DirectionResponse responseForCourt = directionResponseBuilder.build();
        DirectionResponse responseForCafcass = directionResponseBuilder.respondingOnBehalfOf("CAFCASS").build();
```

Reply:
if this can be merged today, no I am not gonna do this. But sure, could be done to cut down lines of cdoe